_episodes/05-mplhep.md

```diff
@@ -106,14 +106,173 @@ Copy and paste these lines to your code
 
 
 ```python
-dy = np.array([0,0,0,0,0,0.354797,0.177398,2.60481,0,0,0,0,0,0,0,0,0,0.177398,0.177398,0,0.177398,0,0,0,0,0,0,0,0,0,0,0,0.177398,0,0,0,0])
-
-ttbar = np.array([0.00465086,0,0.00465086,0,0,0,0,0,0,0,0.00465086,0,0,0,0,0,0.00465086,0,0,0,0,0.00465086,0.00465086,0,0,0.0139526,0,0,0.00465086,0,0,0,0.00465086,0.00465086,0.0139526,0,0])
+dy = np.array(
```

Comment:
Hmm, I'm not a big fan of having so much hard coded data here anyway, but I think maybe we should format this because else it will take soooo much space.

Could you reset these "data fields" and add `fmt: off` to the beginning of these cells and `fmt: on` at the end? This will disable black on them

Reply:
You could simply revert the last commit from pre-commit, then do these changes and push again.

Reply:
I have made the changes. Thanks.